src/openzaak/templates/admin/nlx_config.html

```diff
@@ -0,0 +1,61 @@
+{% extends "admin/base_site.html" %}
+{% load i18n static %}
+
+{% block extrastyle %}{{ block.super }}
+    <link href="{% static 'libs/prism/prism.css'%}" rel="stylesheet" />
+
+{% endblock %}
+
+
+{% block title %} {% trans "Importeer catalogus" %} {{ block.super }} {% endblock %}
+
+{% block breadcrumbs %}
+<div class="breadcrumbs">
+<a href="{% url 'admin:index' %}">{% trans 'Home' %}</a>
+&rsaquo; <a href="#">{% trans 'Configuratie' %}</a>
+&rsaquo; {% trans 'NLX configuratie' %}
+</div>
+{% endblock %}
+
+{% block content %}
+    <h1>{% trans 'NLX configuratie' %}</h1>
+    <div id="content-main">
+        <form action="" method="POST" enctype="multipart/form-data">
```

Comment:
Why is this a form? There's nothing to submit, no?

Reply:
That was me, I now remember that was to submit some variables to change the config file, but removed it later. I'm not sure if removing it will break the layout but if not can be removed.

Reply:
form is removed. Looks ok for me